crates/extensions/c8y_log_manager/src/actor.rs

```diff
@@ -352,7 +350,7 @@ impl LogManagerActor {
 
 #[async_trait]
 impl Actor for LogManagerActor {
-    type MessageBox = LogManagerMessageBox;
+    type MessageBox = SimpleMessageBox<LogInput, NoMessage>;
```

Comment:
Although there's nothing wrong with this, one might wonder why this is not a  `SimpleMessageBox<LogInput, MqttMessage>`.

Reply:
This can be done but would require to change the code as currently the `Sender<MqttMessage>` is owned by the actor and not its message box.

Reply:
We can fix this later - as we are currently considering to break the concept of message box into two concepts (Sender + Receiver) and to merge the Actor and MessageBox concepts.